src/gui_tool/gui/sp_gui.py

```diff
@@ -129,8 +131,8 @@ def handle_keyboard(self, key_mapper: KeyMapper):
             sm.split(i)
         elif key_mapper.consume("z"):
             sm.erase_split(i)
-        elif key_mapper.consume("t"):
-            sm.toggle_section(i)
+        # elif key_mapper.consume("t"):
```

Comment:
why is this removed? Should we be doing something else to cancel a selection?

Reply:
The canceling would be done by adding a cancelling tag

Reply:
so can we no longer cancel in the splitter? Or can we add tags to segments in the splitter now?

Reply:
The ability to add tags to segments has been around for a while. With this change, that'll be the only way to cancel segments

Reply:
The key to do so is v